guides/common/modules/con_http-booting.adoc

```diff
@@ -3,4 +3,16 @@
 [id="http-booting"]
 = HTTP booting
 
-You can use HTTP booting to boot systems over a network using HTTP.
+The term HTTP booting refers to two booting stages _UEFI HTTP Boot_ and _Boot Loader HTTP Boot_.
+
+UEFI HTTP booting is the process of fetching the _Network Bootstrap Program_ (NBP) by the UEFI firmware using HTTP(S) protocol.
+The NBP is typically the boot loader such as GRUB2.
+
+Boot loader HTTP booting is the process of fetching installation files such as kernel and initrd by the boot loader using HTTP(S) protocol.
```

Comment:
```suggestion
Boot loader HTTP booting is the process of fetching installation files such as kernel and initrd by the boot loader over HTTP(S).
```

I am unsure if my suggestion is still technically correct; it also applies to line 8